tests/arch/arm/arm_interrupt/testcase.yaml

```diff
@@ -11,3 +11,12 @@ tests:
       - CONFIG_NO_OPTIMIZATIONS=y
       - CONFIG_IDLE_STACK_SIZE=512
       - CONFIG_MAIN_STACK_SIZE=1024
+  arch.interrupt.extra_exception_info:
+    filter: CONFIG_ARMV6_M_ARMV8_M_BASELINE or CONFIG_ARMV7_M_ARMV8_M_MAINLINE
+    tags: arm interrupt ignore_faults
+    arch_allow: arm
+    extra_configs:
+      - CONFIG_NO_OPTIMIZATIONS=y
```

Comment:
why do we need to disable optimizations to test this?

Reply:
no reason, just copied from the config above. I'll remove all the extra config settings except `CONFIG_EXTRA_EXCEPTION_INFO=y` and push an update